src/Option.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { getOptionLabel, getOptionValue } from './helpers';
+
+const Option = props => {
+  const { option, selected, onSelect, className } = props;
+  const value = getOptionValue(option);
+  const label = getOptionLabel(option);
+  const isSelected = value === selected;
+  let optionClassName = `Dropdown-option ${className || ''}`;
```

Comment:
import classNames from 'classnames';

```suggestion
  const optionClassName = classNames('Dropdown-option', className, { 'is-selected': isSelected });
```